data-prepper-core/src/test/java/com/amazon/dataprepper/parser/config/DataPrepperAppConfigurationTest.java

```diff
@@ -115,4 +117,14 @@ public void testGivenInvalidPipelineConfigArgThenExceptionThrown() throws IOExce
 
         assertThrows(IllegalArgumentException.class, () -> appConfiguration.dataPrepperConfiguration(dataPrepperArgs, objectMapper));
     }
+
+    @Test
+    public void testPluginModelFromDataPrepperConfigurationAuthentication() {
+        final DataPrepperConfiguration configuration = mock(DataPrepperConfiguration.class);
+
+        final Optional<PluginModel> optional = appConfiguration.pluginModel(configuration);
+
+        assertThat(optional.isPresent(), is(false));
+        verify(configuration, times(1)).getAuthentication();
```

Comment:
FYI, `verify` already works only for one time. So you don't need `times(1)` - it is the default behavior.